tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,33 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
+		givenLoggedInUser(before);
+
+		before(function () {
+			this.appName = 'SampleOrg/sampleApp';
+			return balena.models.application
+				.generateApiKey('SampleOrg/sampleApp')
+				.then(function (apiKey) {
+					expect(_.isString(apiKey)).to.be.true;
+					expect(apiKey).to.have.length(32);
```

Comment:
Mostly I want to use the `a('string')`, but combining it into one sentence is a bit nicer overall imo
```suggestion
					expect(apiKey).to.be.a('string').that.has.length(32);
```